### PR TITLE
Update MLP dependency references

### PR DESCRIPTION
**Pull request by terryyylim**
**What this PR does / why we need it**:

`mlp` dependency has been moved under `caraml-dev` organisation via https://github.com/caraml-dev/mlp and this PR updates Turing to reflect that change.

**Which issue(s) this PR fixes**:

`N/A`

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: infra/docker-compose/dev/docker-compose.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ services:
       - migrate -path=/db-migrations -database=postgres://mlp:mlp@mlp-postgres:5432/mlp?sslmode=disable up
 
   mlp:
-    image: ghcr.io/gojek/mlp:v1.1.0-alpha
+    image: ghcr.io/caraml-dev/mlp:v1.7.5-build.5-59f13e1
```
Comment (krithika369) [inline infra/docker-compose/dev/docker-compose.yml:126]: Should we cut a v1.7.6 on MLP to update these refs to proper release versions?

**Review by krithika369 — APPROVED** [review]
Left one small question - not necessary to address in this PR. Thanks, @terryyylim !

File: infra/docker-compose/dev/docker-compose.yml (hunk 1 above)
Comment (terryyylim) [reply infra/docker-compose/dev/docker-compose.yml:126]: Updated!

**Review by deadlycoconuts — APPROVED** [review]
LGTM! :D Thanks for updating the dockerfile!

**Pull request merged**